csfieldguide/templates/interactives/mips-assembler.html

```diff
@@ -79,7 +89,7 @@ <h3>{% trans 'Assembler Output' %}</h3>
   </div>
 
   <!-- Hidden Text: Basic Example Code -->
-  <div id="basic-example" style="display: none;">.data
+  <div id="basic-example" class="d_none">.data
```

Comment:
is this class supposed to be `d-none` not `d_none` ? Same in the simulator?

Reply:
yes, good spot